### DISCUSSION TITLE
How can i create menu item for zooming?

### DISCUSSION THREAD

**Discussion by fk1blow** [tauri-apps/tauri#2715] [Q&A]
Tried using `CmdOrControl+=` or `CmdOrControl+-`, even `CmdOrControl+` or `CmdOrControl=`, but i keep seeing 
`"Cannot map key {:?}" = "Cannot map key {:?}"` in my console. Even tried with a shift modifier, but still cannot 
make it work.
Need to mention that i searched through both issues and discussions and couldn't find anything related to this.

edit: found a native menu item for this, tho i suppose it only works on OSX

What is the correct way to implement zoom in/out?

**Comment by mike-luabase** [comment]
@fk1blow could you share your solution?
